godotproject/Tiles/starryNight/starryNight.tscn

```diff
@@ -0,0 +1,52 @@
+[gd_scene load_steps=5 format=2]
+
+[ext_resource path="res://Tiles/starryNight/starrynight-0.png" type="Texture" id=1]
+[ext_resource path="res://Tiles/starryNight/comet1.png" type="Texture" id=2]
+[ext_resource path="res://Tiles/starryNight/starryNight.gd" type="Script" id=3]
+
+[sub_resource type="Animation" id=1]
+resource_name = "comet"
+length = 8.0
+loop = true
+tracks/0/type = "value"
+tracks/0/path = NodePath("CometSprite:position")
+tracks/0/interp = 1
+tracks/0/loop_wrap = false
+tracks/0/imported = false
+tracks/0/enabled = true
+tracks/0/keys = {
+"times": PoolRealArray( 0, 1, 3.1, 3.7, 4, 4.8 ),
+"transitions": PoolRealArray( 1.5, 1, 1.5, 1, 1.5, 1 ),
+"update": 0,
+"values": [ Vector2( 640, -80 ), Vector2( 48, 592 ), Vector2( 200, -80 ), Vector2( -80, 200 ), Vector2( 720, 100 ), Vector2( 228, 592 ) ]
+}
+tracks/1/type = "value"
+tracks/1/path = NodePath("CometSprite:visible")
+tracks/1/interp = 1
+tracks/1/loop_wrap = true
+tracks/1/imported = false
+tracks/1/enabled = true
+tracks/1/keys = {
+"times": PoolRealArray( 0, 1, 3.1, 3.7, 4, 4.8 ),
+"transitions": PoolRealArray( 1, 1, 1, 1, 1, 1 ),
+"update": 1,
+"values": [ true, false, true, false, true, false ]
+}
+
+[node name="StarryNightScene" type="Node2D"]
+script = ExtResource( 3 )
+
+[node name="BGSprite" type="Sprite" parent="."]
+scale = Vector2( 1.5, 1.5 )
```

Comment:
```suggestion
```
This should be removed, it isn't necessary. If you're having issues with the background not showing up properly on the credits screen then you need to reimport the assets since the asset itself should tile properly now.
Just for reference, to reimport all assets:
1. close godot
2. pull changes
3. delete godotproject/.import folder
4. reopen godot